app/views/userreports/_form.html.erb

```diff
@@ -1,22 +1,22 @@
-<%= form_with model: userreport do |form| %>
+<%= form_with url:  profile_userreports_path(params[:profile_id]) do |form| %>
 
     <%= form.label "Please select a reason:"%>
 
       <%= form.text_field :reporter_id, value: current_user.id %>
 
-      <%= form.radio_button :reason, 'BUY', :checked => true %> 
+      <%= form.radio_button :reason, 'Fake account', :checked => true %> 
       <%= form.label 'Fake account'%>
       
-      <%= form.radio_button :reason, 'SELL' %>
+      <%= form.radio_button :reason, 'Fake name' %>
       <%= form.label 'Fake name' %>
 
-      <%= form.radio_button :reason, 'SELL' %>
+      <%= form.radio_button :reason, 'Pretending to be someone' %>
       <%= form.label 'Pretending to be someone' %>
 
-      <%= form.radio_button :reason, 'SELL' %>
+      <%= form.radio_button :reason, 'Posting inappropriate things' %>
       <%= form.label 'Posting inappropriate things' %>
 
-      <%= form.radio_button :reason, 'SELL' %>
+      <%= form.radio_button :reason, 'Suspicious or spam' %>

```

Comment:
May back-end validation ba 'to? or purely front-end yung pagstate ng reason?

Reply:
purely front-end lang nilagay ko po

Reply:
I think it would be reasonable na may common reasons na selections tas may "others" if none of the select options are applicable